### PR TITLE
AI holocall hangups

### PR DESCRIPTION
**Pull request by optimumtact**
\>Be AI
\>Holocall to my vault
\>Constant ringing sound
\>no escape
\>ringing
\>ringing
\>I want it to end
\>ringing
\>let me die
\>ringing
\>ringing

The AI can now hangup holocalls

:cl: oranges
add: AI's can now hang up all holocalls at a station with alt+click
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
@Cyberboss let me know if there is a more elegant way to achieve this

**Comment by Cyberboss** [comment]
Nah, this is fine

**Comment by optimumtact** [comment]
I added a check when processing so invalid holocalls auto hangup, the issue was that the connected holopad was gone or null, so you could not hang up the call due to runtimes

**Comment by Iamgoofball** [comment]
>playing AI ever

**Comment by Cyberboss** [comment]
This is annoying though because the caller ID supposed to clean up if
something goes wrong. I wonder why that isn't happening

**Comment by optimumtact** [comment]
Stripped out my fix as 27801 does it better

File: code/game/machinery/hologram.dm [hunk 1]
```diff
@@ -113,6 +113,10 @@ Possible to do for anyone motivated enough:
 		return ..()
 
 /obj/machinery/holopad/AltClick(mob/living/carbon/human/user)
```
Comment (KorPhaeron) [inline code/game/machinery/hologram.dm]: carbon/human/user seems wrong

Comment (optimumtact) [inline code/game/machinery/hologram.dm]: oh it is, but it doesn't have any affect since nothing in the proc assumes a specific type

**Pull request merged**